src/main/java/org/jenkinsci/plugins/github/util/JobInfoHelpers.java

```diff
@@ -0,0 +1,60 @@
+package org.jenkinsci.plugins.github.util;
+
+import com.cloudbees.jenkins.GitHubRepositoryName;
+import com.cloudbees.jenkins.GitHubRepositoryNameContributor;
+import com.google.common.base.Function;
+import com.google.common.base.Predicate;
+import hudson.model.AbstractProject;
+import hudson.triggers.Trigger;
+
+import java.util.Collection;
+
+/**
+ * Utility class which holds converters or predicates (matchers) to filter or convert job lists
+ *
+ * @author lanwen (Merkushev Kirill)
+ * @since TODO
+ */
+public final class JobInfoHelpers {
+
+    private JobInfoHelpers() {
+        throw new IllegalAccessError("Do not instantiate it");
+    }
+
+    /**
+     * @param clazz trigger class to check in job
+     *
+     * @return predicate with true on apply if job contains trigger of given class
+     */
+    public static Predicate<AbstractProject> withTrigger(final Class<? extends Trigger> clazz) {
+        return new Predicate<AbstractProject>() {
+            public boolean apply(AbstractProject job) {
+                return job.getTrigger(clazz) != null;
+            }
+        };
+    }
+
+    /**
+     * Can be useful to ignore disabled jobs on reregistering hooks
+     *
+     * @return predicate with true on apply if job is buildable
+     */
+    public static Predicate<AbstractProject> isBuildable() {
```

Comment:
No, <code>Job</code> exists in Jenkins for a long-long time. So it's better to use it in util methods wherever it's possible in order to simplify further integrations when and if they happen.